### ISSUE TITLE
page d'acceuil - hamburger

### ISSUE THREAD

**Issue by nicomq** [scdantilles/angole-theme#2]
sur certaines résolutions mobile, le bouton menu hamburger passe par-dessus le titre.
Screenshot sur iphone 5 (mais ok sur android nexus 5 par ex)
![img_0573-1](https://cloud.githubusercontent.com/assets/10815686/12628243/99b50f14-c518-11e5-841c-d70631ceee27.PNG)

**Comment by kivutar** [comment]
Oui, il nous faut deux types de titre, titre long et titre court, d'après ce que je vois sur les captures. Je vais aussi retravailler le header pour que le menu hamburger soit vraiment à sa place, plus à droite, et pour réduire la marge à droite du logo.

**Comment by kivutar** [comment]
Je viens de pousser les changements au niveau CSS. Ca devrait déjà améliorer la situation.

**Comment by nicomq** [comment]
Effectivement c'est mieux. On pourrait aussi : 
- ou bien simplement réduire la string à "BU Guadeloupe"
- ou bien avoir un `<span class="superflu">Les </span> BU <span class="superflu">de </span> Guadeloupe` et n'afficher le superflu qu'en grand écran.

**Comment by kivutar** [comment]
J'avais pensé a utiliser les helpers bootstrap pour ça oui. Par contre comme c'est le titre du site, il est inclu dans plein d'endroits qui ne supportent pas le markup.

**Comment by nicomq** [comment]
Dans ce cas je propose de simplement raccourcir le titre : BU Guadeloupe
